packages/reactivity/src/effect.ts

```diff
@@ -141,11 +141,11 @@ export function track(target: object, type: TrackOpTypes, key: unknown) {
     return
   }
   let depsMap = targetMap.get(target)
-  if (depsMap === void 0) {
```

Comment:
I think it only want to check the deps map is not undefined, about void 0: https://github.com/lessfish/underscore-analysis/issues/1